testsuite/devserver.go

```diff
@@ -82,7 +81,7 @@ type DevServerOptions struct {
 
 // Temporal CLI based DevServer
 type DevServer struct {
-	cmd    *exec.Cmd
+	Cmd    *exec.Cmd
```

Comment:
not a fan of making it public, though this seems to be the only option to test the RC after interrupting the server (especially for Windows)

Reply:
unless we don't test this and only rely on `/cli`'s tests